css/styles.css

```diff
@@ -677,3 +677,49 @@ textarea:focus {
   padding-left: 10px;
   margin-top: 20px;
 }
+
+.menu {
+  background: #3c3a39;
+  color: #fff5e1;
+  font-size: 25px;
+  font-family: "Inter", sans-serif;
+  font-weight: 700;
+  padding: 20px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: calc(100vw - 19px);
+  height: 100vh;
+}
+
+.menu-items ul li {
+  margin-bottom: 25px;
+  padding-bottom: 10px;
+  border-bottom: 1px solid #6f6c6b;
+  list-style: none;
+}
+
+.menu-items ul li a {
+  color: #fff5e1;
+  text-decoration: none;
```

Comment:
- [ ] Please remove the color blue and replace it with white color instead and also after clicking on the `a` tag it turns to purple and that's not suitable for view.

![Capture web_16-11-2021_202151_localhost](https://user-images.githubusercontent.com/61562626/142051547-86d790d4-0981-4a63-b2ac-fb0e79c67c08.jpeg)